### PR TITLE
Working defs

### PR DESCRIPTION
**Pull request by Coladel**
Functions of CRUD now are defs on code.

### CODE REVIEW

File: crud.rb [hunk 1]
```diff
@@ -1,32 +1,65 @@
 require './create'
+require './read'
+require './delete'
+require './update'
 
-puts 'CRUD do João, Gerencie aqui seus posts'
+      puts
```
Comment (lucasrenan) [inline crud.rb]: 💅  identation

**Comment by lucasrenan** [comment]
@Coladel split code into different pull requests for each crud action

File: read.rb [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+      puts r
+      end
+
+res.clear
+conn.close
+end
+
+def read_last_one
+conn = PGconn.connect('localhost', 5432, nil, nil, 'posts', 'postgres', 'mude6363')
+res = conn.exec("select * from posts where id = (select max(id) from posts)")
```
Comment (lucasrenan) [inline read.rb]: `SELECT * FROM posts ORDER BY id DESC LIMIT 1`

File: read.rb [hunk 3]
```diff
@@ -1,11 +1,36 @@
-puts "Leia as mensagem já inclusas!"
-	
 require 'pg'
+
+def read_individual(value)
 conn = PGconn.connect('localhost', 5432, nil, nil, 'posts', 'postgres', 'mude6363')
-res = conn.exec("select * from posts")
+res = conn.exec("select post from posts where id = '#{value}'")
```
Comment (lucasrenan) [inline read.rb]: `SELECT post FROM posts WHERE id = #{value}`

File: crud.rb [hunk 4]
```diff
@@ -1,32 +1,65 @@
...
+            end
+      end
 
-  create(message)
-end
+      if action =='3'
+        puts "Se deseja atualizar a ultima mensagem digite 0"
+        puts "Caso contrário informe o id da mensagem que deseja atualizar"
+        id_message = gets.chomp
+            if id_message == '0'
```
Comment (lucasrenan) [inline crud.rb]: remove conditional, user can only update passing an id.

File: crud.rb [hunk 5]
```diff
@@ -1,32 +1,65 @@
...
+      puts '4 - Deletar Post'
 
-puts 'Digite o número referente a opção que deseja utilizar!'
-puts
-puts '1 - Criar Post'
-puts '2 - Ler Post'
-puts '3 - Alterar Post'
-puts '4 - Deletar Post'
+      puts 'Informe a opção:'
+      action = gets.chomp
```
Comment (lucasrenan) [inline crud.rb]: `action = user_action`

```
# actual menu.rb
def user_action
  puts "..."

  gets.chomp
end
```

File: crud.rb (hunk 1 above)
Comment (Coladel) [reply crud.rb]: Done!

File: crud.rb (hunk 5 above)
Comment (Coladel) [reply crud.rb]: now menu is called from crud (only with instructions) and variable action is on crud.rb

File: read.rb (hunk 2 above)
Comment (Coladel) [reply read.rb]: Postgres selection corrected, now select by order desc with limit one

File: read.rb (hunk 3 above)
Comment (Coladel) [reply read.rb]: removed extras ' '

File: crud.rb (hunk 4 above)
Comment (Coladel) [reply crud.rb]: now update only works passing an id.

File: crud.rb (hunk 1 above)
Comment (lucasrenan) [reply crud.rb]: looks like it's still wrong, but don't worry I'll fix it for you

**Pull request merged**